### ISSUE TITLE
User cannot use keyboard navigation to navigate to app bar (Safari)

### ISSUE THREAD

**Issue by sairina** [learningequality/kolibri#9183]
## Observed behavior
Using a keyboard to attempt to navigate after signing in is not possible in Safari.

![2022-03-14 11 53 24](https://user-images.githubusercontent.com/13563002/158241436-c90a2370-a269-4328-92b7-acf2a5e3f94b.gif)

## Errors and logs
None

## Expected behavior
To be able to use keyboard navigation in Safari

## User-facing consequences
Users that need to use the keyboard and only have Safari will not be able to navigate through the app

## Steps to reproduce
1. Log in
2. Click on `Tab` to navigate after logging in. Notice that you will see the "Skip to main navigation" option, but even after clicking, you will be unable to access the rest of the nav bar or the main content.

## Context
Kolibri 0.15
MacOS
Safari

**Comment by marcellamaki** [comment]
Possibly Mac settings issue 
![safari-tab-main-nav.gif](https://images.zenhubusercontent.com/6047f6e691103a78d3ad0ace/b8367a21-57fb-4c71-a64b-7f863489cb90)

**Comment by pcenov** [comment]
Confirmed - with the correct settings enabled the tab navigation works correctly on Safari, Firefox and Chrome on Mac.
